### PR TITLE
Chart integration support

### PR DESCRIPTION
**Pull request by Harkishen-Singh**
Signed-off-by: Harkishen-Singh <harkishensingh@hotmail.com>
Follows #44 

The base for chart-js integration in a modular format.

Screenshot from 2019-11-06 21-37-19

### CODE REVIEW

File: src/components/layouts/BRCharts.tsx [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+  constructor(props: BRChartProps) {
+    super(props);
+    console.log('props is');
+    console.log(this.props.opts);
+    this.state = {
+      chartJSOpts: null,
+      wrapperClassName: 'canvas-chart-wrapper'
+    };
+  }
+
```
Comment (ganeshpatro321) [inline src/components/layouts/BRCharts.tsx]: @Harkishen-Singh Its better to use a prop "type" for this component, and then we can use conditional statements to render charts. (type here can be "line". "bar", etc)

Comment (Harkishen-Singh) [reply src/components/layouts/BRCharts.tsx]: @ganeshpatro321 sure. Will do as said.

Comment (Harkishen-Singh) [reply src/components/layouts/BRCharts.tsx]: @ganeshpatro321 actually, we plan to use as a line chart only. time-series values are best suited for line-charts (also area charts, can be converted by `fill: true` property). Hence. the `className` is just used to provide a wrapper style to the component, and hence a state.

Defining as a prop would require the parent component to sent the className which will always be same for every screen, and hence a redundancy. What are your views?

Comment (ganeshpatro321) [reply src/components/layouts/BRCharts.tsx]: Well if we are sure that we will only use line charts, then its good to go. @Harkishen-Singh

**Comment by ganeshpatro321** [comment]
Rest looks good, merge this, we can implement the same for rest of the  modules

**Comment by Harkishen-Singh** [comment]
@ganeshpatro321 sure. Will merge once the end-to-end integration of ping is done. Should happen within a few hours. Thanks.

**Pull request merged**